pom.xml

```diff
@@ -49,7 +49,7 @@
         <dep.accumulo.version>1.10.2</dep.accumulo.version>
         <dep.accumulo-hadoop.version>2.7.7-1</dep.accumulo-hadoop.version>
         <dep.antlr.version>4.11.1</dep.antlr.version>
-        <dep.airlift.version>225</dep.airlift.version>
+        <dep.airlift.version>226-SNAPSHOT</dep.airlift.version>
```

Comment:
Don't forget this one

Reply:
Yeah, this PR is for testing. I'll merge and release Airlift after the Trino CI passes.